### DISCUSSION TITLE
video playback - meshcentral

### DISCUSSION THREAD

**Discussion by t9omek** [Ylianst/MeshCentral#5775] [General]
Hi, 
I would like to watch a video from meshcentral. So far I have been using the website 
https://meshcentral.com/player.htm
Unfortunately the site is currently unavailable. 
Has the service been moved to another site?

**Comment by si458** [comment]
yes the meshcentral.com was running on VM but it got shutdown and moved to github
so the player.htm file doesnt exist anymore,
i will look at seeing if we can put the player.htm back onto github to help people who need to use it
BUT in the mean time if you have your own meshcentral server, you can access the player via `https://YOURMESHDOMAIN.COM/player.htm`

**Reply by ksingh-scogo** [reply]
@si458  it would be a great help if you / project maintainers can put the `player.htm` back onto github.

**Reply by si458** [reply]
@ksingh-scogo it already is, it's in the repo, https://github.com/Ylianst/MeshCentral/blob/master/views/player.handlebars
But it's designed to be used with meshcentral so you will have to download missing files and patch it manually
